package.json

```diff
@@ -80,5 +80,9 @@
     },
     "coverageDirectory": "../coverage",
     "testEnvironment": "node"
-  }
+  },
+  "engines" : 
+  {
+     "node" : ">=0.10.3 <0.12"
```

Comment:
should we not have from 12 as well so we force v12?

Reply:
that's what was suggested yesterday